include/openthread/link.h

```diff
@@ -1127,6 +1157,26 @@ otError otLinkSetEnabled(otInstance *aInstance, bool aEnable);
  */
 bool otLinkIsEnabled(otInstance *aInstance);
 
+/**
+ * This function indicates whether or not the link layer is csl enabled.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ *
+ * @retval true   Link layer is csl enabled.
+ * @retval false  Link layer is not csl enabled.
+ *
+ */
+bool otLinkIsCslEnabled(otInstance *aInstance);
+
+/**
+    * This function indicates whether the device is connected to a parent which supports CSL.
+    *
+    * @retval TRUE   If parent supports CSL.
+    * @retval FALSE  If parent does not support CSL.
+    *
+    */
```

Comment:
Invalid format. We can format all the source files by `./script/make-pretty clang-format` at OT root directory.